shared/js/media/media_utils.js

```diff
@@ -19,6 +21,75 @@ var MediaUtils = {
     });
   },
 
+  formatExifDate: function(dateString) {
+    var [y, m, d] = dateString.split(exifDateSplitRe, 3);
+
+    return new Date(y, m - 1, d).toLocaleString(navigator.languages, {
+      'month': 'numeric',
+      'year': 'numeric',
+      'day': 'numeric'
+    });
+  },
+
+  // exposure is a fraction
+  formatExifExposure: function(exposureTime) {
+    if (!exposureTime || exposureTime.numerator === undefined ||
+        exposureTime.numerator === 0 ||
+        exposureTime.denominator === undefined ||
+        exposureTime.denominator === 0) {
+      return null;
+    }
+    if (exposureTime.numerator > exposureTime.denominator) {
+      return (exposureTime.numerator / exposureTime.denominator).toString();
+    }
+    // trivial fraction simplification.
+    var num = exposureTime.numerator;
+    var den = exposureTime.denominator;
+    if (den % num === 0) {
+      den = den / num;
+      num = 1;
+    }
+    return num + '/' + den;
+  },
+
+  formatExifFlash: function(flashValue) {
+    var FLASH_FIRED_MASK = 0x1;
+//    var FLASH_RETURN_MASK = 0x6;
+    var FLASH_MODE_MASK = 0x18;
+//    var FLASH_FUNCTION_MASK = 0x20;
+    var FLASH_REDEYE_MASK = 0x40;
```

Comment:
I left them because they are 1. the "spec" 2. if not commented the linter complain. :-/